README.md

```diff
@@ -0,0 +1,28 @@
+# Mender Application Updates
+
+The repo contains Mender Update Modules for supporting containerized application updates on devices.
```

Comment:
```suggestion
The repo contains Mender Update Module and artifact generator
```

Reply:
@merlin-northern what are the plans for future? Will the repo contain only those or it's a plans for something more?

Reply:
at one point we will provide support for applications based on something else than containers. for now we focus on container-based deployments.